src/Sarif/Autogenerated/Run.cs

```diff
@@ -5,6 +5,7 @@
 using System.CodeDom.Compiler;
 using System.Collections.Generic;
 using System.Runtime.Serialization;
+using Microsoft.CodeAnalysis.Sarif.Readers;
```

Comment:
The changes to the autogenerated files `Run.cs` and `RunEqualityComparer.cs` are the result of changing the schema to give the `run` object a property bag.